documentation/Functions.md

```diff
@@ -35,10 +35,10 @@ cobiGenJava("path/to/java/file/MyEntity.java",[1,3,5,6,8])
 
 ### createProject
 #### parameter 
-1. the programming language that you will use for this project
+1. the template used for project generation
 2. the project name
 
 #### example 
-createProject("java","cobigenexample")
+createProject("devon4j","cobigenexample")
```

Comment:
I even thought of creating a step for it as there will be no logic at all to be reused. So we should split this into createDevon4jProject("cobigenexample") and further